packages/debugger/test/data/returnvalue.js

```diff
@@ -322,7 +322,7 @@ describe("Return value decoding", function () {
     let instance = await abstractions.ReturnValues.deployed();
     let txHash;
     try {
-      await instance.failNoisy(); //web3 throws on failure
+      await instance.failNoisy({ gas: gasLimit }); //web3 throws on failure
```

Comment:
We need to do this because previously it would fail on tx execution and now it will fail on gas estimation, correct?

Reply:
Yup, exactly.  Putting in `gas` because we want the tx to happen and get reverted, rather than not to happen at all.